### PR TITLE
Test Python 3.13

### PR DESCRIPTION
**Pull request by hugovk**
Pending CFFI support for 3.13, should be available in CFFI 1.17.0 soonish:

* https://github.com/python-cffi/cffi/issues/105

### CODE REVIEW

**Comment by hugovk** [comment]
CFFI 1.17.0 is out, Python 3.13 now passes ✅ 

https://github.com/python-cffi/cffi/releases/tag/v1.17.0

**Review by ezio-melotti — COMMENTED** [review]
Should we add a couple of comments to these two files to remind people to update both?

**Comment by hugovk** [comment]
I don't think it's needed, but no strong opinion.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.11", "3.12", "3.13"]
```
Comment (ezio-melotti) [inline .github/workflows/ci.yml:15]:
```suggestion
        # remember to update the envlist in tox.ini too
        python-version: ["3.11", "3.12", "3.13"]
```
Maybe this comment is enough.

**Review by ezio-melotti — APPROVED** [review]

**Pull request merged**